apps/data-pipeline/course-scraper/src/index.ts

```diff
@@ -571,6 +572,317 @@ async function patchH32(meta: {
   }
 }
 
+async function fetchSchoolsPath(catalogue: string) {
+  const html = await fetchWithDelay(`${catalogue}/schoolsandprograms/`);
+  const $ = load(html);
+  const paths: string[] = [];
```

Comment:
in general, you should be a tad more verbose with variable names and/or comments; what are these paths pointing to?

scraping code in general can be difficult to read but let's try to alleviate that